staging/src/k8s.io/api/core/v1/annotation_key_constants.go

```diff
@@ -56,12 +56,6 @@ const (
 	// in the Annotations of a Node.
 	PreferAvoidPodsAnnotationKey string = "scheduler.alpha.kubernetes.io/preferAvoidPods"
 
-	// SysctlsPodAnnotationKey represents the key of sysctls which are set for the infrastructure
-	// container of a pod. The annotation value is a comma separated list of sysctl_name=value
-	// key-value pairs. Only a limited set of whitelisted and isolated sysctls is supported by
-	// the kubelet. Pods with other sysctls will fail to launch.
-	SysctlsPodAnnotationKey string = "security.alpha.kubernetes.io/sysctls"
```

Comment:
I guess this doesn't need to maintain backwards compatibility since it's alpha? The release note should at least be "action required" to draw attention to this breakage.

Reply:
That's my assumption. I will update the release notes.

Reply:
Will `Action required: All sysctl annotation in existing deployments needs to be converted to API fields.` be sufficient? Is there any prescribed form the action required item?

Reply:
i discussed this with @liggitt .  i know a number of users have used this feature in its alpha form, but migrating the support for both annotations and fields is a land mine.  i think the closest parallel is tolerations which also worked as annotations, went to first class fields, and appears to not support annotations anymore.

Reply:
@ingvagabund the unsafe systctl pods annotation key should also be removed, no?